lib/pbench/server/selinux.py

```diff
@@ -0,0 +1,24 @@
+# This is a mock file for selinux.restorecon test cases
+import os
+
+
+def restorecon(path, recursive=False, verbose=False, force=False):
+    """ Restore SELinux context on a given path
+
+    Arguments:
+    path -- The pathname for the file or directory to be relabeled.
+
+    Keyword arguments:
+    recursive -- Change files and directories file labels recursively (default False)
+    verbose -- Show changes in file labels (default False)
+    force -- Force reset of context to match file_context for customizable files,
+    and the default file context, changing the user, role, range portion  as well
+    as the type (default False)
+    """
+    restorecon_flags = False
+    status = selinux_restorecon(os.path.expanduser(path), restorecon_flags)
+    return status
+
+
+def selinux_restorecon(pathname, restorecon_flags):
```

Comment:
When not running the unit tests, how does this work?

Reply:
yeah, that's what I am working on.. just pushed the work.

Reply:
just wanted to see whether it is working on Jenkins majorly

Reply:
I believe this has been resolved.